lib/glific/flows/flow_context.ex

```diff
@@ -227,14 +229,37 @@ defmodule Glific.Flows.FlowContext do
     parent_id = get_in(opts, [:event_meta, :parent_id])
     source = Keyword.get(opts, :source, "")
 
+    event_label =
+      cond do
+        !is_nil(message) && source == "reset_all_contexts" ->
+          "Flow terminated abruptly"
+
+        !is_nil(parent_id) ->
+          "Child Flow Completed"
+
+        true ->
+          "Flow Completed"
+      end
```

Comment:
I can see that we are using this function in multiple places. Can you write a new funcation with message and source and use that ?

Reply:
@pankaj-ag this is getting used in two places.  I can make a separate function, but the issue is that the condition expression evaluates the different conditions in both places. I think it would be better to keep them for right now. Let me know your thoughts on it.